cmd/minishift/cmd/root.go

```diff
@@ -71,6 +76,11 @@ var RootCmd = &cobra.Command{
 	Short: "Minishift is a tool for application development in local OpenShift clusters.",
 	Long:  `Minishift is a command-line tool that provisions and manages single-node OpenShift clusters optimized for development workflows.`,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		// Prevent creating home dir for specific commands with "minishift" as its parent command.
+		if minishiftStrings.Contains(preventHomeDirCreationForCmds, cmd.Name()) && cmd.Parent().Name() == "minishift" {
```

Comment:
@budhrg we should have variable for `"minishift"` and then use it e.g. `minishiftParantCmd := "minishift"` Because in future if we need to check it in other places we can reuse the variable. It will look more clean too.

Reply:
that is constant then

Reply:
@budhrg yes, it is.